native/acter/src/api/client.rs

```diff
@@ -951,16 +949,16 @@ impl Client {
     }
 
     pub async fn verified_device(&self, dev_id: String) -> Result<bool> {
-        let c = self.core.client().clone();
+        let client = self.core.client().clone();
         let user_id = self.user_id()?;
         RUNTIME
             .spawn(async move {
-                let dev = c
+                client
                     .encryption()
                     .get_device(&user_id, device_id!(dev_id.as_str()))
                     .await?
-                    .context("client should get device")?;
-                Ok(dev.is_verified())
+                    .context("client should get device")
```

Comment:
We should make the error more understandable, I'd say.
```suggestion
                    .context("Unable to find device")
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1248/commits/dcabd9c2c0230aeb14a5336afa87f21c48051d79